lib/mirage_key.ml

```diff
@@ -156,6 +156,12 @@ let warn_error =
   let key = Arg.(opt ~stage:`Configure bool false doc) in
   Key.create "warn_error" key
 
+let gdb =
+  let doc = "Start a gdb server (only available for ukvm)." in
```

Comment:
This ("Start a gdb server") is not actually what the option does. Per the comments on the PR, we decided that `-g` or `--gdb` should be a generic option to "enable (unspecified) debugger support for this target".  I'd suggest rewording this to something like:

"Enables target-specific support for debugging. Supported targets: ukvm (compiles ukvm-bin with GDB server support)."

Also, given the change to a more generic "enable debugging" description for this option, should we call it only `-g` and drop all the references to GDB? This would include renaming the key internally to something more generic (target_debug?)